gslb/cache/controller_obj_cache.go

```diff
@@ -321,14 +327,19 @@ type GSMember struct {
 	Controller string
 }
 
+type HealthMonitorDetails struct {
+	HealthMonitorNames       string
```

Comment:
Does `HealthMonitorDetails` contain only health monitor? If so, why is the field named `HealthMonitorNames` instead of `HealthMonitorName`? Better yet, you can just change it to `Name`.